_includes/navbar.html

```diff
@@ -52,6 +52,16 @@
                         <li><a href="https://cwiki.apache.org/confluence/display/JCLOUDS/How+to+Contribute+Documentation">How To Contribute Documentation</a></li>
                     </ul>
                 </li>
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">About Apache<strong class="caret"></strong></a>
+                    <ul class="dropdown-menu">
+                        <li><a href="http://www.apache.org">The Apache Software Foundation</a></li>
+                        <li><a href="http://www.apache.org/foundation/contributing.html">Donations</a></li>
+                        <li><a href="http://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
+                        <li><a href="http://www.apache.org/security/">Security</a></li>
```

Comment:
Just because it describes the procedure to report security vulnerabilities, and I've seen it added there in several Apache project pages. Perhaps a link in the "report a bug" page should be added too.